saleor/graphql/order/mutations/orders.py

```diff
@@ -386,6 +388,7 @@ def perform_mutation(cls, _root, info, **data):
             app=info.context.app,
             message=cleaned_input["message"],
         )
+        transaction.on_commit(lambda: info.context.plugins.order_updated(order))
```

Comment:
We discussed it with @patrys, and decided to do it in this way, as it's updating the order history.

Reply:
This mutation can be also triggered for draft order, so we should move this logic here as well:
```
        if order.status == OrderStatus.DRAFT:
            func = info.context.plugins.draft_order_updated
        else:
            func = info.context.plugins.order_updated
```

I'm thinking that we could actually use this logic for all order mutations, so that we are sure that we'll always call the right webhook. Maybe we should create a small helper function so that we don't have to copy these four lines in every mutation.